src/main/java/org/opensearch/plugin/insights/core/service/TopQueriesService.java

```diff
@@ -536,4 +540,45 @@ private void drain() {
     public TopQueriesHealthStats getHealthStats() {
         return new TopQueriesHealthStats(this.topQueriesStore.size(), this.queryGrouper.getHealthStats());
     }
+
+    /**
+     * Validate the exporter delete after value
+     *
+     * @param deleteAfter exporter and reader settings
+     */
+    static void validateExporterDeleteAfter(final int deleteAfter) {
+        if (deleteAfter < MIN_DELETE_AFTER_VALUE || deleteAfter > MAX_DELETE_AFTER_VALUE) {
+            OperationalMetricsCounter.getInstance().incrementCounter(OperationalMetric.INVALID_EXPORTER_TYPE_FAILURES);
```

Comment:
We should use another type instead of reusing `INVALID_EXPORTER_TYPE_FAILURES`

Reply:
On second thought, I am in favor of removing the counter here. This is just validation on a user provided cluster setting value. A spike would only tell us that someone repeatedly provided an invalid input. It does not indicate any defect in the system.